Frontend/CO.CDP.OrganisationApp/Pages/YourDetails.cshtml.cs

```diff
@@ -60,6 +60,12 @@ public async Task<IActionResult> OnPost()
             return Page();
         }
 
+        var personInviteId = session.Get<Guid?>("PersonInviteId");
+        if (personInviteId != null)
+        {
+            return RedirectToPage("ClaimOrganisationInvite", new { personInviteId });
+        }
```

Comment:
@dharmverma I've also added the redirect to the registration flow, which takes place after the privacy policy and name screens.